android/build.gradle

```diff
@@ -47,8 +47,6 @@ dependencies {
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
     implementation 'com.github.nurture-farm.clevertap-android-sdk:clevertap-android-sdk:4.0.1x'
-//    implementation 'com.android.support:support-v4:28.0.0'
-    implementation 'com.android.installreferrer:installreferrer:1.0'
```

Comment:
why is this commented out?

Reply:
I have just removed line 50 as it was previously commented out by @shubhendu-nurture at commit: 69636dd43771d8421a8e09380350b28e4862bafc

I have removed line 51 as the codebase has no direct dependency on installReferrer api.